tests/snowpark/test_snowpark_shared.py

```diff
@@ -21,15 +21,14 @@ def test_snowpark_package(
         True,
         SplitRequirements([Requirement.parse("another-package-in-anaconda")], []),
     )
-    Path("app.py").touch()
+    os.mkdir("app")
+    Path(os.path.join("app", "app.py")).touch()
```

Comment:
Something like this could work. 

```python
app_root = Path("app")
app_root.mkdir()
app_root.joinpath(Path("app.py")).touch()
```

`app_root` can be reused later.

Reply:
Done